growth/too/views.py

```diff
@@ -564,6 +564,16 @@ class PlanManualForm(ModelForm):
         validators=[validators.DataRequired()],
         default='REPLACE ME')
 
+    submitter = TextField(
+        validators=[validators.DataRequired()],
+        default='REPLACE ME')
+
+    subprogram_name = TextField(
```

Comment:
I don't understand what this field is for. Perhaps some help text?

Reply:
Added.